### DISCUSSION TITLE
airsdk.dev - no direct link to AIR SDK Download @ Harman

### DISCUSSION THREAD

**Discussion by 2jfw** [airsdk/Adobe-Runtime-Support#2720] [Ideas]
Just saw that **airsdk.dev** does not have a direct link to the Harman download page (https://airsdk.harman.com/download) of the AIR SDK so just a suggestion to maybe add a **Download** item in the upper main menu (left) next to **Docs** ?

**Comment by B-Interactive** [comment]
There are download links in the Getting Started section for each respective operating system:
https://airsdk.dev/docs/basics/getting-started

But perhaps you meant a direct link specifically on the main navbar?

**Reply by marchbold** [reply]
Yeah the idea of the airsdk.dev site was to give a complete portal and eventually not rely on the harman site. So we hadn't planned any direct links. 

The plan is to use the AIR SDK Manager to download the SDK (will likely be looking at a command line version as well for automation).

**Reply by marchbold** [reply]
Though we do need to add the runtime downloads and details on licenses etc. So not quite there yet.
